saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -4,6 +4,14 @@ setup_ccmv2() {
   : ${CCM_V2_AGENT_KEY_ID:? required}
   : ${CCM_V2_AGENT_BACKEND_ID_PREFIX:? required}
 
+ACCESS_KEY_PATH="/etc/ccmv2/access_key"
+touch $ACCESS_KEY_PATH
```

Comment:
See my earlier comment. This aborts because the parent directory does not exist.